src/Components/Home/Home.js

```diff
@@ -9,24 +10,35 @@ import './Home.css';
 
 const Home = ({ favorites }) => {
   const [owls, setOwls] = useState([])
+  const [error, setError] = useState('')
 
   const fetchOwls = useCallback(async () => {
```

Comment:
This is a new one for me on line 13, that looks like such a nice way to define async functions within components!